requirements-app.txt

```diff
@@ -1,76 +1,99 @@
-# This is used to create Postgres stored functions via the migrations process.
```

Comment:
Do not change this file.  Copy the backup we created back to this file, and delete the backup.

Reply:
Did not change requirements-app.txt file